### ISSUE TITLE
Add vga666 mode 6

### ISSUE THREAD

**Issue by vince87** [raspberrypi/firmware#1018]
Hello, can you add the vga666 mode 6 overlay in the future release?
Thanks
Vincenzo

**Comment by pelwell** [comment]
Can you explain what you mean by this? There is already a `vga666` overlay (as I think you know), but all it does is set the pin functions for GPIOs 2-21 to enable the DPI interface - the firmware manages everything else.

**Comment by popcornmix** [comment]
Possibly the 666 format mentioned here: https://github.com/fenlogic/vga666/blob/master/documents/vga_manual.pdf which uses a different set of gpio lines.

**Comment by pelwell** [comment]
That would make sense. Assuming that is the case, try this [updated vga666 overlay](https://drive.google.com/file/d/157IuzZdv8kZB8bYbXI1bsJ2jtOcX_AnH/view?usp=sharing):
1. Download [the file](https://drive.google.com/file/d/157IuzZdv8kZB8bYbXI1bsJ2jtOcX_AnH/view?usp=sharing) and copy it to /boot/overlays/vga666.dtbo
2. Modify your config.txt to include `dtoverlay=vga666,mode6` and reboot.

**Comment by 6by9** [comment]
Also in our DPI documentation - https://www.raspberrypi.org/documentation/hardware/raspberrypi/dpi/README.md
mode 6 would imply GPIOs 4-9, 12-17, and 20-25.

**Comment by vince87** [comment]
@popcornmix , yes, it's correct, I would like to use mode 6 because it leaves 2 gpio free for audio (13,18).
@pelwell in the next few days, i try your overlay!

**Comment by JamesH65** [comment]
@vince87 Did you try this? Did it work? Can this issue be closed?
